### PR TITLE
Added initial contribution documentation

### PR DESCRIPTION
**Pull request by paulanunda**
Added documentation on how to contribute to the marketplace-kubernetes repo.

### CODE REVIEW

File: utils/generate-stack.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+# create stack directory
+NEW_DIR_PATH="$ROOT_DIR/stacks/$STACK_NAME"
+mkdir "$NEW_DIR_PATH"
+
+# copy files
+cp -r "$ROOT_DIR"/utils/stack-templates/* "$NEW_DIR_PATH"
+
+for file in "$NEW_DIR_PATH"/*
```
Comment (scottcrawford03) [inline utils/generate-stack.sh]: why are we putting the yaml into the stack? doesn't this mean that if we want to use this yaml in any other stack will have to also have it in that stack, thus causing duplication all over? my understanding was that we'd have one place were the yaml lived and reference it from within a stack. that way if we update the yaml we do it in one place, versus doing it in each stack that we create.

Comment (paulanunda) [reply utils/generate-stack.sh]: There's src YAML (typically included as part of a Helm template, or raw k8s manifests), and stack (rendered) YAML. Rendered meaning it has either been customized using Kustomize, Helm values, or both.

**Comment by scottcrawford03** [comment]
Makes sense! Thanks for the clarification!

File: utils/generate-stack.sh (hunk 1 above)
Comment (scottcrawford03) [reply utils/generate-stack.sh]: 👍

**Comment by scottcrawford03** [comment]
LGTM even though it's already merged.

**Pull request merged**